tests/test_configs.py

```diff
@@ -55,3 +58,45 @@ def test_architecture_init_name_only():
     assert architecture.extra_configs == []
     assert architecture.fragments == []
     assert architecture._filters == []  # filters does not have a property..
+
+
+class ConfigTest(unittest.TestCase):
+
+    def _load_config(self, yaml_file_path):
+        with open(yaml_file_path) as yaml_file:
+            ref_data = yaml.safe_load(yaml_file)
+        config = kernelci.config.load(yaml_file_path)
+        return ref_data, config
+
+    def _reload(self, ref_data, config, name):
+        assert name in config
```

Comment:
Erm, that's actually not how you write assertions for `unittest.TestCase` based tests. There are special methods to do that https://docs.python.org/3/library/unittest.html#basic-example

Reply:
Maybe this can work if you use pytest as a runner (haven't checked), but even if you write something which is pytest-only then it shouldn't extend `unittests.TestCase`

Reply:
Yes, sorry I just stuck to the `pytest` approach even though this is `unittest` (run inside `pytest`).

Reply:
It's still misleading to use `unittest.TestCase` here, especially that the whole `test_configs.py` is written in a pytest convention and is not meant to be run with unittest test runner.
Having said that, I'm all for using pytest, which has a lot of advantages over unittest.

Reply:
Yes I think I was more interested in reusing code from a base class than using `unittest`.  So I'll see if I can rework this with classes and play `pytest`, maybe it's just the case of dropping the `TestCase` inheritance since there's no `setUp()` or `tearDown()`.

Reply:
OK I've reworked this without `unittest`, the base class name just has to start with `Test` as per the naming convention used by `pytest` to discover tests.